modules/javafx.swing/src/main/java/javafx/embed/swing/JFXPanel.java

```diff
@@ -148,6 +148,8 @@ public class JFXPanel extends JComponent {
     private transient volatile EmbeddedWindow stage;
     private transient volatile Scene scene;
 
+    private final Object LOCK = new Object();
+
     // Accessed on EDT only
```

Comment:
is this comment valid anymore?

perhaps we should mention the fact that access to these fields should be LOCK'ed instead?